include/zephyr/dt-bindings/flash_controller/npcx_fiu_qspi.h

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2023 Nuvoton Technology Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_NPCK_FIU_QSPI_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_CLOCK_NPCK_FIU_QSPI_H_
+
+#include <zephyr/dt-bindings/dt-util.h>
+
+/**
+ * @name NPCX QSPI definition for configuring the Quad-SPI peripherals
+ * For example:
+ *
+ * int_flash: w25q40@0 {
+ *      compatible ="nuvoton,npcx-fiu-nor";
+ *          status = "okay"
+ *          reg = <0>;
+ *          size = <0x400000>;
+ *          qspi-flags = <(NPCX_QSPI_SW_CS0 | NPCX_QSPI_SEC_FLASH_SL)>;
+ *  };
+ */
```

Comment:
I would not bother with this example comment block, there's already an example in the yaml file, this will just go stale.

Reply:
I will remove the redundant comment later. :)

Reply:
Done.